pib-rocks/pib-backend
# Merge develop into main

# Jira Ticket:
Link to corresponding jira ticket:
https://pib-rocks.atlassian.net/jira/software/c/projects/PR/boards/3?selectedIssue=PR-622

Branches without ticket:
- "update_pull_request_template"


# Checklist:
Necessary tasks before pull request can be merged:
- [x] All necessary tests have been successfully conducted

# Testing:
### Testing approach for setups:
- [x] Setup on a physical pib was successful

### Testing acceptance criteria:

- [x] PR-622: New audio files are used for turning the voice assistant on and off (the new files are more responsive)

### Testing instructions:

- Run the setup script with parameters -b=develop -f=develop
- PR-622: turn the voice assistant on and off and check if the new audio files can be heard
